### ISSUE TITLE
[Bug]: mattermost-desktop package grew from 280 MB to 1172 MB in installed size between 5.9.0 and 5.10.0 due to addition of debug info

### ISSUE THREAD

**Issue by xtaran** [mattermost/desktop#3208]
### Checks before filing an issue

- [X] This issue doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/mattermost-server/issues).
- [X] I have checked the [issue tracker](https://github.com/mattermost/desktop/issues) and have not found an issue that matches the one I'm filing.
- [X] This issue is not a troubleshooting question. Troubleshooting questions go here: https://forum.mattermost.com/c/trouble-shoot/16.
- [X] This issue is not a feature request. You can request features and make product suggestions here: https://mattermost.com/suggestions/.
- [X] This issue reproduces on the most recent [stable version](https://github.com/mattermost/desktop/releases/latest), or the most recent [prerelease version](https://github.com/mattermost/desktop/releases) of the Mattermost Desktop App.
- [X] I have read the [contribution guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

### Mattermost Desktop Version

5.10.0

### Operating System

Debian GNU/Linux 12.8 amd64

### Mattermost Server Version

not relevant, issue present before running mattermost-desktop

### Steps to reproduce

```
[/var/cache/apt/archives] # ls -lhv mattermost-desktop_5.*
-rw-r--r-- 1 root root  75M Aug 15 22:33 mattermost-desktop_5.9.0_amd64.deb
-rw-r--r-- 1 root root 177M Nov 15 17:01 mattermost-desktop_5.10.0_amd64.deb
[/var/cache/apt/archives] # apt-cache show mattermost-desktop=5.9.0 mattermost-desktop=5.10.0 | egrep 'Package:|Version:|Installed-Size|^$'
Package: mattermost-desktop
Installed-Size: 273379
Version: 5.9.0

Package: mattermost-desktop
Installed-Size: 1144839
Version: 5.10.0
```

5.9.0:

```
$ ls -lh /opt/Mattermost/mattermost-desktop
-rwxr-xr-x 1 root root 174M Aug 15 22:22 /opt/Mattermost/mattermost-desktop
```

5.10.0:

```
$ ls -lh /opt/Mattermost/mattermost-desktop
-rwxr-xr-x 1 root root 1.1G Nov 15 16:51 /opt/Mattermost/mattermost-desktop
```

### Expected behavior

Moderate size growth, not by _factor_ 4.

### Observed behavior

Sudden massive increase of disk space usage on root or /opt partition.

### Log Output

```shell
# df -h /
Filesystem              Size  Used Avail Use% Mounted on
/dev/mapper/vgssd-root   26G   22G  3.0G  89% /
# dpkg -i mattermost-desktop_5.10.0_amd64.deb 
(Reading database ... 675096 files and directories currently installed.)
Preparing to unpack mattermost-desktop_5.10.0_amd64.deb ...
Unpacking mattermost-desktop (5.10.0) over (5.9.0) ...
Setting up mattermost-desktop (5.10.0) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for mailcap (3.70+nmu1) ...
Processing triggers for gnome-menus (3.36.0-1.1) ...
Processing triggers for desktop-file-utils (0.26-1) ...
# df -h /
Filesystem              Size  Used Avail Use% Mounted on
/dev/mapper/vgssd-root   26G   23G  2.1G  92% /
#
```

### Additional Information

5.9.0:

```
$ file /opt/Mattermost/mattermost-desktop 
/opt/Mattermost/mattermost-desktop: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=fa86c7b143cef28859310fe826b76b793998cc8f, not stripped
```

5.10.0:

```
$ file /opt/Mattermost/mattermost-desktop 
/opt/Mattermost/mattermost-desktop: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=c233fc5898b4d6cac0e32c0f1d9a0735dafb5068, with debug_info, not stripped
```

Debug information belongs into a separate package to be downloaded only by those who actually need it…

**Comment by devinbinnie** [comment]
This only seemed to affect the x64 package, the ARM package seems fine. I wonder if this was caused by an `electron-builder` change.

Created a ticket to investigate: https://mattermost.atlassian.net/browse/MM-61833

**Comment by xtaran** [comment]
> This only seemed to affect the x64 package, the ARM package seems fine. I wonder if this was caused by an `electron-builder` change.

That these is caused by Electron's build setup is also my suspicion as I didn't find any (other) related commit in Mattermost Desktop's code than the bump of the Electron version used to build it. 

> Created a ticket to investigate: https://mattermost.atlassian.net/browse/MM-61833

Thanks!

Version 5.10.1 is affected, too, btw.

**Comment by xtaran** [comment]
Seems fixed with 5.10.2, thanks!
